src/nfsv4mountscript.sh

```diff
@@ -24,14 +24,22 @@ CERT_PATH=
 CERT_UPDATE_COMMAND=
 STUNNEL_CAFILE=
 
+# Temporary mountmap file used to overwrite mountmap file.
+TMP_MOUNTMAPv4="/tmp/mountmapv4.tmp"
```

Comment:
are you sure they will always have this /tmp folder?

Reply:
Yes, but I can change it to concatenate the moutnmap instead of using the tmp file.